docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -30,14 +30,137 @@ Understanding GBD's Disease and Health Categories
+in the context of a specific example. We will use cardiovascular disease for this. 
+In particular, we will examine: 
 
-- Showing what is a direct subset, what overlaps, what interacts but is independent 
-- Include some information on modeling implications as a result of this 
+- Causes: ischemic heart disease (IHD) and hypertensive heart disease (HHD) 
+- Health states: medically managed, mild, moderate and severe heart failure 
+- Impairments: heart failure 
+- Sequelae: numerous, including "severe heart failure due to IHD" and "AMI first 2 days" among others 
 
+To help with understanding some of the below information, a diagram of the heart failure impairment is provided: 
+
+.. image:: health_state_impairment_sequela.svg
+
+
+Causes
+++++++
+
+The cause hierarchy diagram for IHD and HHD is included here as a reference. 
+
+.. image:: cause_hierarchy_ihd_hhd.svg
+
+As you can see, the cause hierarchy slowly breaks down cause groups into more specific 
+diseases, moving from all non-communicable diseases to specific types of cardiovascular 
+disease. At each level (labeled), the cause hierarchy is mutually exclusive and 
+collectively exhaustive - meaning all disease and death can be placed into a group at 
+each level of the diagram. A cause is anything that causes death or disability. 
+
+Both IHD and HHD are within the more general cardiovascular disease cause (c_491). IHD and HHD 
+are "causes" within GBD. 
+
+Sequelae
+++++++++
+
+Once you have selected a specific cause - for example, IHD - you can then identify the 
+sequelae within that cause. These are all of the items listed below IHD and HHD in the 
+diagrams above. 
+
+Sequelae detail a specific medical condition associated with 
+a cause. For example, IHD has medical conditions like acute myocardial infarction 
+(a heart attack) and angina (chest pain) as associated sequelae. These are the things 
+a patient might be experiencing, or a doctor might identify. 
+
+Sequelae often contain a level of severity for that condition - for example, mild and moderate 
+angina are both sequelae within IHD. 
+
+Health States
++++++++++++++
+
+A health state is the amount of health or disability within a sequelae. This is summarized 
+through the disability weight. More information on this can be found in `this section of the cause model page <https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_model_components/causes/index.html#disability-weights>`_. A health state for a sequelae and its disability weight can be 
+found in the GBD methods appendix. 
+
+Using our above sequelae (acute myocardial infarction and angina) as an example - 
+the amount of health lost while experiencing an acute 
+myocardial infarction is quite high! You're likely hospitalized and in a lot of pain. 
+However, the health lost from angina might be less significant. You're likely at home and 
+might only experience discomfort when under stress. This affects your disability weight value. 
```

Comment:
Do you show the health states and disability weights associated with acute myocardial infarction and angina anywhere? The first sentence here made me think you were referring to specific numbers for the disability weights, but I don't see them. And I'm also curious what the descriptions of the health states for these sequelae are -- are they basically the same as the sequela names?

Reply:
I had not included these but added them here.